bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/system/TransitionalSPTarget.java

```diff
@@ -0,0 +1,75 @@
+package edu.gemini.spModel.target.system;
+
+import edu.gemini.shared.skyobject.Magnitude;
+import edu.gemini.shared.util.immutable.ImList;
+import edu.gemini.spModel.target.SpatialProfile;
+import edu.gemini.spModel.target.SpectralDistribution;
+import edu.gemini.spModel.target.WatchablePos;
+
+// transitional; will go away
```

Comment:
I often mark temporal classes as `@Deprecated`, the compiler warnings motivate me to take them out